python/ray/dashboard/client/src/pages/dashboard/node-info/features/Disk.tsx

```diff
@@ -1,14 +1,17 @@
 import { Typography } from "@material-ui/core";
 import React from "react";
 import { formatUsage } from "../../../../common/formatUtils";
+import { Accessor } from "../../../../common/tableUtils";
 import UsageBar from "../../../../common/UsageBar";
 import {
-  ClusterFeatureComponent,
-  NodeFeatureComponent,
-  WorkerFeatureComponent,
+  ClusterFeatureRenderFn,
+  NodeFeatureData,
+  NodeFeatureRenderFn,
+  NodeInfoFeature,
+  WorkerFeatureRenderFn,
 } from "./types";
 
-export const ClusterDisk: ClusterFeatureComponent = ({ nodes }) => {
+export const ClusterDisk: ClusterFeatureRenderFn = ({ nodes }) => {
```

Comment:
Is this used anywhere?

Reply:
Good catch. This is supposed to be exported in the object at the bottom.